src/lib/urls.js

```diff
@@ -30,3 +30,39 @@ export function getCanonicalPath(path) {
   }
   return parts.join('/');
 }
+
+export function copyLinkToClipboard() {
+  if (!('clipboard' in navigator)) {
+    return;
+  }
+  document
+    .querySelector('main')
+    // ToDo: Use `:is(h2, h3, h4, h5, h6)[id]` once support is better.
+    .querySelectorAll('h2[id], h3[id], h4[id], h5[id], h6[id]')
+    .forEach((heading) => {
+      heading.addEventListener('click', (ev) => {
+        // Don't jump when the '#' is clicked.
+        if (ev.target.nodeName === 'A') {
+          ev.preventDefault();
+        }
```

Comment:
I'm not sure why we'd want to prevent this? I looked at css-tricks and it allows the jump.

Reply:
Preventing the jump is pretty common best practice since it avoids user confusion. FWIW, the implementation on WebFu ([sample link](https://developers.google.com/web/fundamentals/performance/critical-rendering-path#critical-rendering-path)) did this, too.



<img width="418" alt="Screen Shot 2021-02-11 at 16 11 07" src="https://user-images.githubusercontent.com/145676/107656329-586db600-6c84-11eb-9f0b-7b2393d38cc3.png">